utils/cookie.ts

```diff
@@ -1,4 +1,20 @@
-import { getSetCookies, setCookie } from "std/http/cookie.ts";
+import { getCookies, getSetCookies, setCookie } from "std/http/cookie.ts";
+
+export const getFlagsFromCookies = (req: Request) => {
+  const flags = [];
+  const cookies = getCookies(req.headers);
+
+  for (const [key, value] of Object.entries(cookies)) {
+    if (key.startsWith("deco_matcher_")) {
```

Comment:
You should import from deco/block/matchers so you can guarantee that the version always is up to date with cookie name